api/lib/opentelemetry/context/propagation/noop_text_map_propagator.rb

```diff
@@ -7,6 +7,8 @@
 module OpenTelemetry
   class Context
     module Propagation
+      # @api private
```

Comment:
I feel like, if we're making this `@api private`, we should just delete all the method documentation. The documentation on `TextMapPropagator` is mostly pretty similar, and folks looking to implement their own will probably discover that implementation before this one 🤷

Reply:
👍 I removed all documentation for the `NoopTextMapPropagator` (API and SDK) in f0688bff .